server/routerlicious/packages/lambdas-driver/src/document-router/documentContext.ts

```diff
@@ -79,12 +80,27 @@ export class DocumentContext extends EventEmitter implements IContext {
 		// Assert offset is between the current tail and head
 		const offset = message.offset;
 
-		assert(
-			offset > this.tail.offset && offset <= this.head.offset,
-			`${offset} > ${this.tail.offset} && ${offset} <= ${this.head.offset} ` +
-				`(${message.topic}, ${message.partition}, ${this.routingKey.tenantId}/${this.routingKey.documentId})`,
-		);
-
+		try {
+			assert(
```

Comment:
it is generally a bad practice to use exception handling for control flow. I also don't know if it is intentional, but this code also swallows the error. you should move the assert condition to an if, and just do the needful there, rather than use an assert + try catch.

Reply:
Ok, thanks for the suggestion. I can convert this to an if statement